### DISCUSSION TITLE
[Help] - Need time picker with 24 hrs format

### DISCUSSION THREAD

**Discussion by anjalikalsariya** [data-driven-forms/react-forms#1464] [Need help] [answered]
**Description** : We are using the  ant-component-mapper and we have time picker fields in form schema.  In form, time picker showing well with 12hr am pm format. We need format 24hr with no am pm. Ant lib providing that format see the **image 2**. 
But I don't know how we can do by the schema ? like which property we can pass in schema for getting 24hr time picker ? 
Anyone , Can you please help me ?

My schema timepicker field
```
 {
      "name": "DEBUTOP",
      "label": "Uhrzeit des Beginns des Eingriffes",
      "className": "form-item-left",
      "component": "time-picker"
    },
```

Current behaviour (Image 1)
<img width="583" alt="image" src="https://github.com/data-driven-forms/react-forms/assets/13288691/9d4d1d2d-35c9-4eb4-989a-bace40960756">

Require behaviour (Image 2)
<img width="162" alt="image" src="https://github.com/data-driven-forms/react-forms/assets/13288691/c173e322-6acd-43ae-93ff-b9b9a70a830c">

**Comment by rvsia** [comment, marked as answer]
Hey,

same as in the [original component ](https://ant.design/components/time-picker), there is a prop `use12Hours` that controls this behavior. By default it is set to `true`, but you can switch it to `false` and it will show the correct 24h format.

**Reply by anjalikalsariya** [reply]
You are right @rvsia , Thanks.
It is working with 24hrs now.
